client/tests/base.spec.ts

```diff
@@ -1,32 +1,39 @@
-import { test, expect } from '@playwright/test';
-test('test', async ({ page }) => {
+import { test, expect } from "@playwright/test";
+test("test", async ({ page }) => {
   // Go to http://localhost:4500/
-  await page.goto('http://localhost:4500/');
+  await page.goto("http://localhost:4500/");
   // Click [placeholder="Room ID"]
   await page.locator('[placeholder="Room ID"]').click();
   // Fill [placeholder="Room ID"]
-  await page.locator('[placeholder="Room ID"]').fill('test');
+  await page.locator('[placeholder="Room ID"]').fill("test");
   // Click text=Create
-  await page.locator('text=Create').click();
-  await expect(page).toHaveURL('http://localhost:4500/test');
-  // Click [placeholder="Name"]
+  await page.locator("text=Create").click();
+  await expect(page).toHaveURL("http://localhost:4500/test");
+  // Click and fill name.
   await page.locator('[placeholder="Name"]').click();
-  // Fill [placeholder="Name"]
-  await page.locator('[placeholder="Name"]').fill('test');
-  // Click .css-1qfu80a-ValueContainer2
-  await page.locator('.css-1qfu80a-ValueContainer2').click();
-  // Click .css-1qfu80a-ValueContainer2
-  await page.locator('.css-1qfu80a-ValueContainer2').click();
+  await page.locator('[placeholder="Name"]').fill("test");
+
+  /* To do: bg-320
```

Comment:
Since I realized the connection bug wasn't caused by this pr, I'm commenting out the broken tests for now to be fixed in: https://github.com/Amijakan/mochatto/issues/320